.github/workflows/release-build-publish.yml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Build/Publish
+
+on:
+  push:
+    branches: 
+      - release
+    paths:
+      - VERSION
+
+jobs:
+  release-build-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+          ref: release
+
+      - name: Login to Quay.io
+        uses: azure/docker-login@v1
+        with:
+          login-server: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+
+      - name: Login to Docker Hub
+        uses: azure/docker-login@v1
```

Comment:
this might be a dumb question but why azure/docker-login? why not `docker/login-action@v1` ?

Reply:
`docker/login-action` doesn't support logging into multiple repos at once, you have to do it incrementally which is annoying. This way we can log into Dockerhub and Quay.io in one go.

Btw I forgot to mention, the Dockerhub secrets need to be added to this repo before merging this PR.

Reply:
oh, that's cool. using the same username/pwd? How does it repo-hop from docker.io into quay.io?

Also I don't have permissions to add the secrets... maybe @benoitf knows who owns those?

Reply:
> This way we can log into Dockerhub and Quay.io in one go.

but you're logging in twice on line 29 then again on line 36. How is that "one go" ?

Reply:
It's one go in the sense that you can be logged into two repositories simultaneously, otherwise to build + push a single image we'd have to:
- log into Dockerhub
- build
- log into Quay.io
- push

and since we are building 4 images that would be kind of tedious :sweat: